### ISSUE TITLE
[Bug]: PDF hyperlinks clickable on only 1st page

### ISSUE THREAD

**Issue by hlau-square** [puppeteer/puppeteer#9298]
### Bug description

Steps to reproduce the problem:

1. run vanilla/basic puppeteer

```
let browser = await puppeteer.launch({ headless: true });
const page = await browser.newPage();
await page.setContent(html, { waitUntil: ['domcontentloaded', 'networkidle0'] })
const pdf = await page.pdf({ format: "A4" });

fs.createWriteStream('/tmp/generated.pdf').write(pdf);
browser.close();
```

2. set the **html** variable above, to a string that has 2 pages of content, forcing the output PDF to be more than 1 page long. Add hyperlinks `<a href>` everywhere. You can use the example HTML attached below.

3. run the code snippet from step 1. Open the output generated PDF file.

4. notice the hyperlinks on the 1st page are all clickable, but none of the hyperlinks on the 2nd page are clickable. Here is an example of the generated PDF
[generated.pdf](https://github.com/puppeteer/puppeteer/files/10061626/generated.pdf)

Here is the HTML
```
<html><body>
<b>1. Definitions.<br/>
</b>
<br/>
<span style="margin-left:15px"></span>
<b>a. “Accepted Agreement”</b> Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement.
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>b.  Accepted Agreement</b> Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>c. Accepted Agreement</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement.
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>d.  Accepted Agreement</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement 
<a href="https://google.com/gb/en/henry">HenryTest1</a>
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>e. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<span style="margin-left:15px"></span>
<b>f.  Accepted Agreement</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>g.  Accepted Agreement</b> or 
<b>“ Accepted Agreement</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>h.  Accepted Agreement</b> or 
<b> Accepted Agreement</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>i.  Accepted Agreement</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>j.  Accepted Agreement</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>k. “ Accepted Agreement”</b>, 
<b>“ Accepted Agreement”</b> or 
<b>“ Accepted Agreement”</b> or 
<b>“ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>l. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>m. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>n. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<b>
<br/>
</b>
<a href="https://google.com/gb/en/henry">HenryTest2</a>
<b>
    <span style="margin-left:15px"></span>
</b>
<b>o. “ Accepted Agreement”</b> Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>p. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>q. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement.
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>r. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<a href="https://squareup.com/gb/en/henry">HenryTest3</a>
<span style="margin-left:15px"></span>
<b>s. “ Accepted Agreement”</b> and 
<b>“ Accepted Agreement”</b>  Accepted Agreement
<br/>
<br/>
<a href="https://squareup.com/gb/en/henry">HenryTest4</a>
<span style="margin-left:15px"></span>
<b>t. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<a href="https://squareup.com/gb/en/henry">HenryTest5</a>
<span style="margin-left:15px"></span>
<b>u. “ Accepted Agreement”</b>   Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/><a href="https://squareup.com/gb/en/henry">HenryTest6</a>
<br/><a href="https://squareup.com/gb/en/henry">HenryTest7</a>
<a href="https://squareup.com/gb/en/henry">HenryTest8</a>
<span style="margin-left:15px"></span>
<b>v. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<a href="https://squareup.com/gb/en/henry">HenryTest9</a>
<span style="margin-left:15px"></span>
<b>w. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<span style="margin-left:15px"></span>
<b>x. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement
<br/>
<br/>
<a href="https://squareup.com/gb/en/henry">HenryTest10</a>
<span style="margin-left:15px"></span>
<b>y. “ Accepted Agreement”</b>  Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement Accepted Agreement.
<br/>
</body>
</html>
```

### Puppeteer version

19.2.2

### Node.js version

6.14.8

### npm version

v14.15.1

### What operating system are you seeing the problem on?

macOS

### Relevant log output

```Shell
no errors in logs
```

**Comment by hlau-square** [comment]
I downgraded from `puppeteer-core` from `19.2.2` to `19.1.2` and the issue seems to be resolved (links on 2nd page are clickable. Can someone please confirm?

**Comment by OrKoN** [comment]
Looks like a regression in Chrome: filed https://crbug.com/1392642

**Comment by OrKoN** [comment]
I believe this should be fixed with the next release (19.6.0) which includes the latest Chromium binary.
